### PR TITLE
Feature/#255 create view all customer orders

### LINKED ISSUE #255: SP: 3 Create view for retrieving all orders of the Customer

### PR DESCRIPTION
**Pull request by Misha86**
Created view for customer orders.
Added sorting and permissions for it.

### CODE REVIEW

File: beauty/api/urls.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from django.urls import path
 
 from api.views.order_views import (OrderApprovingView, OrderListCreateView,
-                                   OrderRetrieveCancelView)
+                                   OrderRetrieveCancelView, CustomerOrdersViews)
```
Comment (Serhii-Voloshyn) [inline beauty/api/urls.py]: Sort imports in alphabetical order

File: beauty/api/views/order_views.py [hunk 2]
```diff
@@ -8,18 +8,18 @@
 from django.utils.encoding import force_str
 from django.utils.http import urlsafe_base64_decode
 
-from rest_framework import status
+from rest_framework import status, filters
```
Comment (Serhii-Voloshyn) [inline beauty/api/views/order_views.py]: Sort imports in alphabetical order

File: beauty/api/views/order_views.py [hunk 3]
```diff
@@ -8,18 +8,18 @@
...
+                                     get_object_or_404, ListAPIView, RetrieveAPIView)
 from rest_framework.permissions import (IsAuthenticated)
 from rest_framework.response import Response
 from rest_framework.reverse import reverse
 from beauty import signals
 from beauty.tokens import OrderApprovingTokenGenerator
 from beauty.utils import ApprovingOrderEmail
-from api.models import Order
-from api.permissions import IsOrderUser
+from api.models import Order, CustomUser
```
Comment (Serhii-Voloshyn) [inline beauty/api/views/order_views.py]: Sort imports in alphabetical order

File: beauty/api/views/order_views.py [hunk 4]
```diff
@@ -8,18 +8,18 @@
...
+from api.permissions import (IsOrderUser, IsCustomerOrders)
```
Comment (Serhii-Voloshyn) [inline beauty/api/views/order_views.py:22]: Sort imports in alphabetical order

**Review by Serhii-Voloshyn — APPROVED** [review]

File: beauty/api/permissions.py [hunk 5]
```diff
@@ -21,11 +20,7 @@ def has_object_permission(self, request, view, obj):
 
         if request.method in permissions.SAFE_METHODS:
             return True
-        return (
-            request.user.is_authenticated and (
-                request.user.is_admin or (obj == request.user)
-            )
-        )
+        return (request.user.is_authenticated and (request.user.is_admin or (obj == request.user)))
```
Comment (morento101) [inline beauty/api/permissions.py:24]: In the last sprint, I rewrote this line because its length was > 79 characters

**Review by morento101 — APPROVED** [review]

**Review by pazuzu-ua — APPROVED** [review]

**Pull request merged**